CI.hs

```diff
@@ -436,12 +436,15 @@ buildDists
       -- 7762 means this shouldn't be an issue anymore going forward.
       ghcOptionsWithHaddock = ghcOptionsOpt
 
-      -- Mitigate macOS w/[ghc-9.0 <= 9.2.2] build failures for lack of
-      -- this c-include path (including e.g. ghc-lib-parser on
-      -- ghc-9.2.2). See
+      -- Mitigate against macOS/ghc-9.2.2 failures for lack of this
+      -- c-include path. See
       -- https://gitlab.haskell.org/ghc/ghc/-/issues/20592#note_391266.
+      -- There are reports that this exhibits with 9.0.2 and 9.2.1 as
+      -- well but I haven't observed that.
```

Comment:
hm from the linked issue it seems that the reports for `9.0.2` and `9.2.1` are from users on M1 macs, I think this would bite us if Microsoft changed the `macOS-latest` image to run on M1, but we know what to fix if it happens